librz/core/cmd_descs/cmd_analysis.yaml

```
@@ -1541,6 +1541,11 @@ commands:
           - RZ_OUTPUT_MODE_RIZIN
           - RZ_OUTPUT_MODE_JSON
         args: []
+      - name: avD
+        summary: mark objects and devirtualize calls to virtual functions
```

```suggestion
        summary: Mark objects and devirtualize calls to virtual functions.
```

For these user facing docs please take are of grammar and spelling.